doc/src/conf.py

```diff
@@ -142,12 +146,96 @@
 html_last_updated_fmt = '%b %d, %Y'
 
 # was classic
-html_theme = "classic"
+# html_theme = "classic"
+html_theme = "furo"
+
+# Adjust the sidebar so that the entire sidebar is scrollable
+html_sidebars = {
+    "**": [
+        "sidebar/scroll-start.html",
+        "sidebar/brand.html",
+        "sidebar/search.html",
+        "sidebar/navigation.html",
+        "sidebar/versions.html",
+        "sidebar/scroll-end.html",
+    ],
+}
 
-html_logo = '_static/sympylogo.png'
-html_favicon = '../_build/logo/sympy-notailtext-favicon.ico'
-# See http://www.sphinx-doc.org/en/master/theming.html#builtin-themes
+common_theme_variables = {
+    # Main "SymPy green" colors. Many things uses these colors.
+    "color-brand-primary": "#81B953",
+    "color-brand-content": "#307748",
+
+    # The left sidebar.
+    "color-sidebar-background": "#3B5526",
+    "color-sidebar-link-text": "#FFFFFF",
+    "color-sidebar-brand-text": "var(--color-sidebar-link-text--top-level)",
+    "color-sidebar-link-text--top-level": "#FFFFFF",
+    "color-sidebar-item-background--hover": "#8AB950",
+    "color-sidebar-item-expander-background--hover": "#8AB950",
+
+    "color-link-underline--hover": "var(--color-link)",
+    "color-api-keyword": "#000000bd",
+    "color-api-name": "#29A329",
+    "color-api-pre-name": "var(--color-brand-primary)",
+    "api-font-size": "var(--font-size--normal)",
+    "color-foreground-secondary": "#53555B",
+
+    # TODO: Add the other types of admonitions here if anyone uses them.
+    "color-admonition-title-background--seealso": "#CCCCCC",
+    "color-admonition-title--seealso": "black",
+    "color-admonition-title-background--note": "#CCCCCC",
+    "color-admonition-title--note": "black",
+    "color-admonition-title-background--warning": "var(--color-problematic)",
+    "color-admonition-title--warning": "white",
+    "admonition-font-size": "var(--font-size--normal)",
+    "admonition-title-font-size": "var(--font-size--normal)",
+
+    # Note: this doesn't work. If we want to change this, we have to set
+    # it as the .highlight background in custom.css.
+    "color-code-background": "hsl(80deg 100% 95%)",
+
+    "code-font-size": "var(--font-size--small)",
+    "font-stack--monospace": 'DejaVu Sans Mono,"SFMono-Regular",Menlo,Consolas,Monaco,Liberation Mono,Lucida Console,monospace;'
+    }
+
+html_theme_options = {
+    "light_css_variables": common_theme_variables,
+    # The dark variables automatically inherit values from the light variables
+    "dark_css_variables": {
+        **common_theme_variables,
```

Comment:
You likely do not need this.

Reply:
If I comment this line out, the light specific modifications do not carry over to dark mode. I think it was working before when I had an older version of Furo but it stopped working when I updated it.

Reply:
Hmm... Fascinating. I'll dig into what's happening here later. I'm pretty sure this won't do the wrong thing though, so this is definitely fine to do anyway. :)